public/javascripts/search-bar.js

```diff
@@ -0,0 +1,15 @@
+const searchBar = document.forms['search-box'].querySelector('input')
+
+searchBar.addEventListener('keyup', function (e) {
+	const term = e.target.value.toLowerCase()
```

Comment:
You're only searching on exact match within the title, not any of the other data points. I'd suggest searching through the full range of available data if possible using a fuzzy strategy